### ISSUE TITLE
Update the runtime

### ISSUE THREAD

**Issue by AsciiWolf** [flathub/io.github.foldynl.QLog#39]
We should consider updating the runtime (and base) version used by this Flatpak to the latest 6.8 one.

This will however require more detailed testing to confirm that all functions still work as expected after the update.

/cc @foldynl

**Comment by foldynl** [comment]
Yes, I already have a patch ready for 6.7, but I haven't tested it on 6.8 yet. I've already made many changes to make it work. I expect to deliver a patch for the next version.

**Comment by AsciiWolf** [comment]
6.8 also contains new fd runtime (24.08) so there may be bigger issues. Be prepared for that and feel free to ping me if you need any help.

**Comment by foldynl** [comment]
I decided to postpone the run-time upgrade due to several bugs that are not yet fixed in version 0.40 and mean more code changes.

**Comment by AsciiWolf** [comment]
Sure, no worries. As an alternative, you can try the `5.15-24.08` runtime that is Qt 5 based.

**Comment by foldynl** [comment]
Done.
